src/main/java/com/frcteam3636/frc2024/subsystems/shooter/ShooterIO.kt

```diff
@@ -82,6 +148,11 @@ class ShooterIOReal : ShooterIO {
         Logger.recordOutput("Shooter/Right Power", rSpeed)
     }
 
+    override fun setPivotControlRequest(control: ControlRequest) {
```

Comment:
We'll need to add gravity feedforward here at some point, as well as possibly make them fight each other cuz of the 90deg gearbox.